src/test/java/util/CommentServiceImplTest.java

```diff
@@ -0,0 +1,32 @@
+package util;
+
+import static org.mockito.Mockito.when;
+
+import com.spring.boot.repository.CommentRepository;
+import com.spring.boot.service.CommentService;
+import com.spring.boot.service.impl.CommentServiceImpl;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class CommentServiceImplTest {
+    private static CommentService commentService;
+    private static CommentRepository commentRepository;
+
+    @BeforeAll
+    static void beforeAll() {
+        commentRepository = Mockito.mock(CommentRepository.class);
+        commentService = new CommentServiceImpl(commentRepository);
+    }
+
+    @Test
+    public void outputListSize_ok() {
+        List<String> tempList = List.of("the ", "and ", "this ", "a ",
+                "it ", "i ", "taffy ", "is ", "was ", "very ", "of ");
+        when(commentService.getMostUsedWordsFromComments(11)).thenReturn(tempList);
```

Comment:
if you are testing the service method, there is no point in mocking this service. Mock repository

Reply:
I do mock repo not service